README.md

```diff
@@ -39,47 +39,79 @@
 ### Get Help or give help
 
 - Open a new [issue](https://github.com/CityOfZion/neo-python/issues/new) if you encounter a problem.
-- Or ping **@localhuman** on the [NEO Slack](https://join.slack.com/t/neoblockchainteam/shared_invite/MjE3ODMxNDUzMDE1LTE1MDA4OTY3NDQtNTMwM2MyMTc2NA).
-- Pull requests welcome. You can help with wallet functionality, writing tests or documentation, or on any other feature you deem awesome.  All successful pull requests will be rewarded with one photo of a cat or kitten.
+- Or ping [@localhuman](https://github.com/localhuman) on the [NEO Slack](https://join.slack.com/t/neoblockchainteam/shared_invite/MjE3ODMxNDUzMDE1LTE1MDA4OTY3NDQtNTMwM2MyMTc2NA).
+- Pull requests are welcome. You can help with wallet functionality, writing tests or documentation, or on any other feature you deem awesome.  All successful pull requests will be rewarded with one photo of a cat or kitten. 
 
+### License
+
+- Open-source [Apache 2.0](https://github.com/CityOfZion/neo-python/blob/master/LICENSE).
+- Main author is [@localhuman](https://github.com/localhuman).
+
+### Project structure
+
+```
+neo-python
+├── compiler.py
+├── prompt.py            <─── terminal client
+├── node.py
+└── neo
+    ├── Core
+        ├── State
+        └── TX
+    ├── Cryptography
+    ├── Implementations
+        ├── Blockchains
+            ├── LevelDB
+            └── RPC
+        └── Wallets
+    ├── IO
+    ├── Network
+        └── Payloads
+    ├── SmartContract
+        ├── Framework
+            ├── Neo
+            └── System
+    ├── Utils
+    ├── VM
+    └── Wallets
+```
 
```

Comment:
I would rather have a docs setup similar to neo-boa to put more extended explanations like this.